cli-tool/cli.py

```diff
@@ -0,0 +1,168 @@
+import json
+import os
+
+import click
+import requests
+from decouple import config
+from rich import print
+from rich.console import Console
+from rich.table import Table
+
+console = Console()
+
+print("Welcome to the CLI Tool of [bold blue]MediaCMS![/bold blue]", ":thumbs_up:")
+
+
+BASE_URL = 'https://demo.mediacms.io/api/v1'
+AUTH_KEY = ''
+USERNAME = ''
+EMAIL = ''
+
+
+def set_envs():
+    with open('.env', 'r') as file:
+        if not file.read(1):
+            print("Use the Login command to set your credential environment variables")
+        else:
+            global AUTH_KEY, USERNAME, EMAIL
+            AUTH_KEY = config('AUTH_KEY')
+            USERNAME = config('USERNAME')
```

Comment:
I've logged in, and see that .env creates the entry USERNAME
However here when I print USERNAME it is set to 'user' - AUTH_KEY and EMAIl are correct from .env

So my-media gives error results (queries `https://demo.mediacms.io/api/v1/media?author=user` instead of `https://demo.mediacms.io/api/v1/media?author=markos` in my case)

Reply:
I am not able to replicate this error. I am getting correct media and username :(